package.json

```diff
@@ -104,7 +104,7 @@
   "eslintConfig": {
     "root": true,
     "parserOptions": {
-      "ecmaVersion": 5
+      "ecmaVersion": 6
```

Comment:
Parser can be changed per folder.
an `.eslintrc.yml` or `eslintrc.json` file in a folder will override project level configuration.